README.md

```diff
@@ -3,7 +3,7 @@
 [![crates.io](https://img.shields.io/crates/v/unleash-edge?label=latest)](https://crates.io/crates/unleash-edge)
 [![Documentation](https://docs.rs/unleash-edge/badge.svg?version=latest)](https://docs.rs/unleash-edge/latest)
 ![MIT licensed](https://img.shields.io/crates/l/unleash-edge.svg)
-[![Dependency Status](https://deps.rs/crate/unleash-edge/19.1.3/status.svg)](https://deps.rs/crate/unleash-edge/19.1.3)
+[![Dependency Status](https://deps.rs/crate/unleash-edge/19.2.0/status.svg)](https://deps.rs/crate/unleash-edge/19.2.0)
```

Comment:
Is this something we need to manually update every time? If so, can we update `<mostrecentversion>` in our quickstart command below to be the latest version as well?

Ideally we would be able to automate both.

Reply:
yes, Ideally we'd like to update this before cutting releases, that way the dependency banner matches the version it belongs to, rather than whenever we remembered to update it last.

I'd love to find a way to update this automatically based on most recent version.